tests/Finder/LocatorTest.php

```diff
@@ -104,6 +104,8 @@ public function test_it_returns_first_possible_file()
         );
 
         $found = $locator->locateAnyOf(['secondfile.txt', 'file.txt']);
+        $found = realpath($found);
```

Comment:
probably, you can wrap it with `p` function as in other tests in this file. It was created to transform window paths to unix-like

see for example:

https://github.com/infection/infection/blob/f59d66736ae089ffefd167e681382d3398655377/tests/Finder/LocatorTest.php#L28

Reply:
Yeah i noticed that as well, should be fixed now.